frontend/src/screens/SingleVideo/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import useCustomContext from '../../customHooks/Hook';
+import { SINGLE_VIDEO_REQUEST, SINGLE_VIDEO_SUCCESS, SINGLE_VIDEO_FAILURE, ADD_TO_HISTORY, ADD_WATCH_LATER, LIKE_VIDEO } from '../../constants/type';
+import axios from 'axios';
+import { Banner, Modal } from '../../components';
+import ReactPlayer from 'react-player';
+import styles from './single_video.module.css';
+import dateformat from 'dateformat';
+
+const SingleVideo = () => {
+
+    const useQuery = () => new URLSearchParams(useLocation().search);
+    const query = useQuery();
+    const videoId = query.get('v');
+    const { state, dispatch } = useCustomContext();
+    console.log({ state })
```

Comment:
You might have missed to remove console here

Reply:
done